src/charm.py

```diff
@@ -92,6 +85,15 @@ def __init__(self, *args):
             getattr(self.on, "log_data_storage_detaching"), self._on_storage_detaching
         )
 
+        self._grafana_agent = COSAgentProvider(
+            self,
+            metrics_endpoints=[
+                # Endpoint for the kafka and jmx exporters
+                # See https://github.com/canonical/charmed-kafka-snap for details
+                {"path": "/metrics", "port": 9101},
```

Comment:
**nitpick/chore:** Could we add `JMX_EXPORTER_PORT=9101` to `literals.py` and f-string here?

Reply:
Done in next commit.